examples/ros-bag-recorder-cpp/readme.txt

```diff
@@ -0,0 +1,10 @@
+
+
+To build proto files:
+
+    cd src/protos
+    sudo protoc -I . --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_cpp_plugin` protos/*/v1/*.proto
+    sudo protoc -I . --cpp_out=. protos/*/v1/*.proto
+
+
+sudo apt install ros-noetic-ros-type-introspection
```

Comment:
is it normal to add 'someone else's protos' to the project like this? what if we checked out the protos from https://github.com/FormantIO/formant/tree/master/protos as part of the build step, and added it to the gitignore

Reply:
I think that is probably a better way to do this